build.gradle

```diff
@@ -34,11 +34,12 @@ dependencies {
 }
 
 mainClassName = 'eu.indigo.jplvalidator.Cli'
-version = '1.1.0'
+version = 'latest'
```

Comment:
I prefer not to do that, it is safer to use an specific version so that they can be easily aligned with the supported jpl versions, e.g.:

- jpl-2.0.0 using jpl-validator-1.0.0
- jpl-2.1.0 using jpl-validator-1.1.0

If we always rely on 'latest', previously working (and supported) jpl versions (e.g. 2.0.0) might stop working if the schema gets upgraded, e.g. as a result of a new mandatory parameter introduced in 3.0.0.

Reply:
I agree if we create branch release/1.1.0 for staging the changes before creating the tag.
We need a label for our development tests and latest can mislead the users.